docs/ht_ww.md

```diff
@@ -1,9 +1,53 @@
 # OVOS Listener - WakeWords / HotWords
-OVOS uses "wakewords" to activate the system.  This is what "hey Google" or "Alexa" is on proprietary devices.  By default, OVOS uses the WakeWord "hey Mycroft".
+OVOS uses "wakewords" to activate the system. This is what "hey Google" or "Alexa" is on proprietary devices. By default, OVOS uses the WakeWord "hey Mycroft".
+
+OVOS "hotwords" is the configuration section in `mycroft.conf` config  to specify what the WakeWord do.  Multiple "hotwords" can be used to do a variety of things from putting OVOS into active listening mode, a WakeWord like "hey Mycroft", to issuing a command such as "stop" or "wake up"
+
+Each hotword can do one or more of the following (with example configs):
+* trigger listening, also called a wake_word (default option)
+  * `“Listen” : “True”`
+
+* play a sound (only or with other action), you can set a specific sound per hotword.
+  * `"sound": "snd/start_listening.wav"`
+
+* emit a utterance to open a skill directly (instead of listen), useful to open skills from others.
+  * `“utterance” : “What’s the time”` 
+  * `“listen” : “false”`
+   
+  This example will start the date & time skill with the given utterance and not listen for a command.
+
+* emit a bus event directly to the messagebus, to for instance open a skill you developed yourself  (instead of listen / utterance), emit a message directly (microphone mute/unmute"
+  *	`“bus_event” : “custom message that you use in your own skill”`
```

Comment:
Incorrect quote characters. Also, consider `custom.message.that.you.use.in.your.own.skill` to match the pattern bus messages uses.